src/helpers/build.js

```diff
@@ -149,7 +151,9 @@ const bundleEs5App = (folder, metadata, options = {}) => {
 
   if (options.sourcemaps === false) args.push('--no-sourcemap')
 
-  return execa(path.join(__dirname, '../..', 'node_modules/.bin/rollup'), args)
+  let currentWorkingDir = process.cwd()
+  let levelsDown = __dirname.indexOf(currentWorkingDir) > -1 ? '../../../../..' : '../..'
+  return execa(path.join(__dirname, levelsDown, 'node_modules/.bin/rollup'), args)
```

Comment:
@sandeep-vedam :
- let's move this into a helper (so we can reuse instead of duplicating)
- the indexOf might need a comment to explain what's happening (or we can do this by naming the variable better, i.e. isLocaInstallation en then based on true / false define how many levels down we should go)